packages/page-council/src/Motions/Motion.tsx

```diff
@@ -81,14 +81,15 @@ function Motion ({ className = '', isMember, members, motion: { hash, proposal,
         votes={nays}
       />
       <td className='button'>
-        {isVoteable && !isCloseable && (
```

Comment:
I figured out this `isClasable` wasn't needed because it's mutually exclusive with `isVotable` afaiu.